service/worker/failovermanager/workflow.go

```diff
@@ -461,3 +486,22 @@ func cleanupChannel(channel workflow.Channel) {
 		}
 	}
 }
+
+func validateTaskListPollerInfo(ctx context.Context, feClient frontend.Client, domain string) error {
+	resp, err := feClient.GetTaskListsByDomain(ctx, &types.GetTaskListsByDomainRequest{Domain: domain})
+	if err != nil {
+		return fmt.Errorf("failed to get task list for domain %s", domain)
+	}
+	for name, tl := range resp.GetDecisionTaskListMap() {
+		if len(tl.GetPollers()) == 0 {
```

Comment:
so we only need to check if there's a poller or not in the target cluster, instead of verifying it for all active task lists in source cluster?

Reply:
The active task lists will be put into the cache in passive matching. So I guess we don't need to

Reply:
Interesting... I thought activity/decision will be pushed to standby matching only when those two tasks are discarded by transfer queue processing logic. Is that what you're referring to, or it's something else?

Reply:
Oh, I see. then we need to query both side.